tests/test_alembic_dialect.py

```diff
@@ -6,3 +7,8 @@
 def test_configure_migration_context():
     context = migration.MigrationContext.configure(url='redshift+psycopg2://mydb')
     assert isinstance(context.impl, dialect.RedshiftImpl)
+
+
+def test_rename_table():
+    compiler = dialect.RedShiftDDLCompiler(dialect.RedshiftDialect(), None)
+    assert compiler.process(RenameTable("old", "new", "scheme")) == 'ALTER TABLE scheme."old" RENAME TO "new"'
```

Comment:
Nice test, but I'd like to avoid horizontal GitHub scrolling for new code.

```python
sql = compiler.process(RenameTable('old', 'new, 'scheme'))
assert sql == 'ALTER TABLE scheme."old" RENAME TO "new"'
```